packages/desktop/views/dashboard/delegation/Delegation.svelte

```diff
@@ -180,7 +183,19 @@
                         isCopyable: true,
                         clearBoxPadding: true,
                         clearBackground: true,
-                        classes: 'text-gray-600 dark:text-white text-xs font-medium',
+                        classes: 'flex flex-row items-center text-gray-600 dark:text-white text-xs font-medium gap-2',
+                    },
+                    slot: {
+                        component: PingingBadge,
```

Comment:
i'm not sure about using this component here... maybe we should ask the rest of the team :pray:  wdyt?

Reply:
I see it well, ask the rest of the team if you want